### PR TITLE
restore edited at timestamp for posts w/tests

### LINKED ISSUE #1697: Post edited timestamp not displaying
Works for comments -- reported by Tibet

### PR DESCRIPTION
**Pull request by KevinTriplett**
Closes #1697 

display edited at timestamp for posts w/tests

### CODE REVIEW

File: src/components/PostCard/PostDetails/PostDetails.js
Comment (tibetsprague) [inline src/components/PostCard/PostDetails/PostDetails.js:1]: Oh wait. I think I was looking in the wrong place! We did want to move the edited text out of the header and show it at the end of the content

**Comment by tibetsprague** [comment]
So you can still see the post/comment creation date. And also to match what other platforms do

**Comment by tibetsprague** [comment]
Sorry about that

**Comment by KevinTriplett** [comment]
Ah! I thought so but couldn't remember -- okay, let me change it back and move the tests, np :)

**Comment by KevinTriplett** [comment]
@tibetsprague I propose putting both "posted" and "edited" timestamps in the post header, because I noticed it's difficult to see the edited timestamp: I have to look for it whereas the posted timestamp is easy to see.

**Comment by KevinTriplett** [comment]
@tibetsprague two commits are optional -- I added an exact timestamp tooltip to the "edited at" timestamp

**Comment by tibetsprague** [comment]
included some of this, and released!

**Pull request closed without merging**